src/HydraAuction/OnChain/Escrow.hs

```diff
@@ -127,17 +127,17 @@ mkEscrowValidator (StandingBidAddress standingBidAddressLocal, FeeEscrowAddress
               Just (StandingBidDatum {standingBidState}) -> case standingBid standingBidState of
                 Nothing -> traceError "No bid in standing bid datum"
                 (Just bidTerms) ->
-                  traceIfFalse "Not signed by bidder" (txSignedBy info (bidBidder bidTerms))
```

Comment:
@GeorgeFlerovsky  Maybe we should check sign validity here too, to prevent possible statebid forgery by L2 peers.